hazelcast-jet-distribution/src/root/release_notes.txt

```diff
@@ -23,6 +23,42 @@ To expand an issue number into a full GitHub link, use this template:
 
 https://github.com/hazelcast/hazelcast-jet/issues/<issue number>
 
+
+** Hazelcast Jet 4.3 **
```

Comment:
Are these all?

* IMDG version is updated
* I can see many other bugs fixed: https://github.com/hazelcast/hazelcast-jet/pulls?q=is%3Apr+milestone%3A4.3+is%3Aclosed+label%3Adefect but only 1 is mentioned here?